math/vector_cross_product.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file vector_cross_product.cpp
```

Comment:
Better not to provide the filename, it will be automatically recognized.
```suggestion
 * @file
```